utilities/functionality_class.py

```diff
@@ -0,0 +1,47 @@
+from selenium.webdriver.firefox.webdriver import WebDriver
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+from constans import emoji
+from utilities.log_class import LogClass
+
+
+class Functionality:
+    driver: WebDriver
+    wait: WebDriverWait
+    log: LogClass
+
+    def __init__(self, driver, wait, log):
+        self.driver = driver
+        self.wait = wait
+        self.log = log
+
+    def find_element(self, element):
+        return self.driver.find_element(*element)
+
+    def click_on_element(self, element):
```

Comment:
rename this method, find_and_click_element, nes tu ne tik clickini

Reply:
fair